lab05/infra/variables.tf

```diff
@@ -4,6 +4,7 @@ variable "environment" {
     condition     = can(regex("^[\\w-]+$", var.environment))
     error_message = "Environment name is not valid."
   }
+  default = "dev"
```

Comment:
To było usunięte celowo, żeby nie tworzyć środowiska na pałę przez przypadkowe uruchomienie - głównie dla mnie kiedy obrabiałem tego laba ;)